webpack.common.js

```diff
@@ -55,15 +55,20 @@ function configure(IS_TEST) {
     module: {
       rules: [
         {
+          test: /settings\.js/,
+          use: [
+            { loader: 'envify-loader' },
+          ],
+        },
+        {
           test: /\.js$/,
           use: [
             { loader: 'cache-loader', options: { cacheIdentifier: CACHE_INVALIDATE } },
             { loader: 'thread-loader', options: { workers: 3 } },
             { loader: 'babel-loader' },
-            { loader: 'envify-loader' },
```

Comment:
@anotherchrisberry @jrsquared any reason not to remove envify here? AFAIK it's only used for `settings.js`

Reply:
I don't see it used anywhere else, either...